Fix and Feat for progress bar

Tbaut opened the pull request:
closes #783 
- use @polkassembly/util@0.2 cf #799 
- adds failing threshold (if the ref is passing)
- adds passing/failing info to make it clear whether or not the referendum is passing
- show the ayes/nays without convictions
- show turnout percentage

![image](https://user-images.githubusercontent.com/33178835/82361738-8da5ea00-9a0b-11ea-8ac1-60b2c0fec38b.png)

![loading](https://user-images.githubusercontent.com/33178835/82350560-cf2e9900-99fb-11ea-8f10-a2cd782f98a1.gif)


todo : more testing

erler commented on front-end/src/ui-components/VoteProgress.tsx:
```diff
@@ -53,17 +51,20 @@ const VoteProgress = ({ ayeVotes, className, nayVotes, passingThreshold }: Props
+					<div
+						className={ thresholdPercent < 50 ? 'threshold-left' : 'threshold-right'}
+					>
+						{isPassing ? 'Failing' : 'Passing'} threshold: {bnToStringBalanceDelimitor(threshold)} {tokenSymbol}
```
I wonder if we should replace the token symbol with "votes" (as well as line 44 and 48). So that `token * conviction = votes`. As you can have more votes than actual issued tokens. And then use the token symbols for the votes without conviction.

Tbaut replied:
I think this might be equally confusing for users. Because it's not defined anywhere that a vote = KSM * conviction. Although it's quite clear I think that users can vote with their KSM and add a conviction. Adding a new terminology means adding some more help tool tips.

erler replied:
This is also true. Removing the token symbol here and adding it to `ReferendumVoteInfo` like you suggested seems like a good solution

erler commented on front-end/src/components/Post/GovernanceSideBar/Referenda/ReferendumVoteInfo.tsx:
```diff
@@ -62,54 +120,53 @@ const ReferendumVoteInfo = ({ className, referendumId, threshold }: Props) => {
+							<Grid.Row>
+								<Grid.Column>
+									<h6>Turnout {turnoutPercentage > 0 && <span className='turnoutPercentage'>({turnoutPercentage}%)</span>}</h6>
+									<div>{formatBnBalance(turnout, { numberAfterComma: 2 })}</div>
```
Wonder if to add the token symbol here and to all votes without conviction (lines 158 and 162).

Tbaut replied:
Sure why now. I'm wondering the amount of text on this screen is getting high. What about actually removing the "KSM" from the progressbar? After all users know we talk about KSM don't you think?

erler replied:
Yeah, adding here and removing from progress bar sound good

erler replied:
Tested and agree with you. Too much text. Let's also remove it from the progress bar

erler commented on front-end/src/ui-components/PassingInfo.tsx:
```diff
@@ -0,0 +1,54 @@
+	border-color: grey_border;
+	border-radius: 3px;
+	margin-bottom: 1rem;
+	font-size: xl;
```
```suggestion
	font-size: lg;
```
looking good :)

erler commented on front-end/src/ui-components/PassingInfo.tsx:
```diff
@@ -0,0 +1,54 @@
+export default styled(PassingInfo)`
+	background-color: grey_secondary;
+	padding: 2rem 3rem 2rem 3rem;
+	border-style: solid;
+	border-width: 1px;
+	border-color: grey_border;
```
No border needed here I think
```suggestion
```

Tbaut commented:
I added KSM in the info section, and removed it from the progress bar. This sounds consistent to me so that we're only talking about KSM when there's no conviction.
